### ISSUE TITLE
Change timer granularity

### ISSUE THREAD

**Issue by xiam** [pressly/imgry#16]

**Comment by pkieltyka** [comment]
Yes. Can you find out if statsd can report the kind of value..? ie. A gauge value for a when we are tracking the "time" it takes to process something should show in sysdig as milliseconds instead of a rate.
